package.json

```diff
@@ -66,7 +66,6 @@
     "@sentry/react": "^7.91.0",
     "@spindl-xyz/attribution-lite": "^1.4.0",
     "@walletconnect/utils": "^2.17.2",
```

Comment:
could this be also taken from reown?

Reply:
I've asked them about this.

Reply:
Looks like they still recommend this package according to their docs: https://docs.reown.com/walletkit/web/installation

Reply:
Although I would suggest to add an explicit dependency to `@walletconnect/core` since we are using it as well.